packages/run-protocol/src/vpool-xyk-amm/pool.js

```diff
@@ -338,15 +340,15 @@ export const definePoolKind = (
   quoteIssuerKit,
   paramAccessor,
   protocolSeat,
+  storageNode,
+  marshaller,
 ) => {
   const poolInit = (liquidityZcfMint, poolSeat, secondaryBrand) => {
     const { brand: liquidityBrand, issuer: liquidityIssuer } =
       liquidityZcfMint.getIssuerRecord();
     const { notifier, updater } = makeNotifierKit();
-    const {
-      publication: metricsPublication,
-      subscription: metricsSubscription,
-    } = makeSubscriptionKit();
+    const { publisher: metricsPublication, subscriber: metricsSubscription } =
```

Comment:
there appears to be a bug now that no childNode is made. I'll try to make a test for that and then fix it.

Reply:
e9e63340c4f6caa6f186e2fef36aef1fe9faf074